openwisp_radius/api/freeradius_views.py

```diff
@@ -445,6 +448,9 @@ def _data_to_acct_model(self, valid_data):
         valid_data['organization'] = acct_org
         return valid_data
 
+    def send_radius_accounting_signal(self, accounting_data):
+        radius_accounting_success.send(sender=self, accounting_data=accounting_data)
```

Comment:
When we are passing `self` from here it is a view object which we can't specify as sender while connecting to receivers.
We must use `__class__` to get the class view and not the view object.
cc: @nemesisdesign

Reply:
@codesankalp we know it's a view instance, the change was done on purpose (django also passes view instances), what is the problem you're trying to describe? I don't understand.

Reply:
Oh I think I understand now, in this case we don't need to take into account the sender though, because only this class will send this. Let's look at how django implements some of their request cycle signals and do it similarly (let's look if they use the view instance or class as sender).

Reply:
@devkapilbansal Sankalp is right, let's do the following which should be more ortodox:

- restore the previous sender (the class)
- pass the request and view instance as an additional argument
- update docs